common/test/acceptance/tests/lms/test_teams.py

```diff
@@ -953,9 +954,12 @@ def test_user_can_navigate_to_members_profile_page(self):
         """
         self._set_team_configuration_and_membership()
         self.team_page.visit()
-
         learner_name = self.team_page.first_member_username
-
+        self.team_page.wait_for_element_visibility(
```

Comment:
Same question here-- we generally don't set a wait time unless we know that the default is not long enough.